test/hotspot/jtreg/runtime/cds/appcds/LambdaWithUseImplMethodHandle.java

```diff
@@ -27,6 +27,8 @@
  * @bug 8290417
  * @summary CDS cannot archive lambda proxy with useImplMethodHandle
  * @requires vm.cds
+ * @requires vm.cds.supports.aot.class.linking
+ * @requires !vm.jvmci.enabled
```

Comment:
Can `@requires` lines have comments? It wouldn't hurt to call out that this is a workaround pending resolution of [JDK-8345969](https://bugs.openjdk.org/browse/JDK-8345969) (or some other issue).

Reply:
I added comments as you suggested.